magprime/site_sections/example.py

```diff
@@ -1,7 +0,0 @@
-from uber.common import *
```

Comment:
Since we have an actual site section now, having an example seems like it's no longer required.